src/lib/stores/billing.ts

```diff
@@ -211,7 +211,10 @@ export async function checkPaymentAuthorizationRequired(org: Organization) {
 
 export async function paymentExpired(org: Organization) {
     if (!org?.paymentMethodId) return;
-    const payment = await sdk.forConsole.billing.getPaymentMethod(org.paymentMethodId);
+    const payment = await sdk.forConsole.billing.getOrganizationPaymentMethod(
```

Comment:
Where is this method used?

Reply:
We use this on load to check if there are any expired payments